.github/workflows/size.yml

```diff
@@ -1,15 +1,17 @@
 name: Compressed Size
 
-on: [pull_request]
+on:
+  pull_request:
+    branches:
+      - '**'
+    paths:
+      - '**/src/**.js'
```

Comment:
I think this glob should do the right thing... but do keep an eye out for the workflow not running when it should